common/src/main/java/tachyon/Constants.java

```diff
@@ -152,8 +152,8 @@
       "tachyon.integration.master.resource.cpu";
   public static final String INTEGRATION_MASTER_RESOURCE_MEM =
       "tachyon.integration.master.resource.mem";
-  public static final String INTEGRATION_YARN_ONE_WORKER_PER_HOST =
-      "tachyon.integration.yarn.one.worker.per.host";
+  public static final String INTEGRATION_YARN_MAX_WORKERS_PER_HOST =
+      "tachyon.integration.yarn.max.workers.per.host";
```

Comment:
can you make it `tachyon.integration.yarn.workers.per.host.max`, to make it consistent with other property names?